README.md

```diff
@@ -64,7 +64,24 @@ To get the source code from GitHub, type
 1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
 It can be installed locally or globally depending on your own needs.
 
-2. Execute the following command in order to get this package:
+2. You can edit your `composer.json` file manually or directly add the requirement via CLI:
+
+###Manually edit
+Create or edit your **composer.json** in the root of your project and add the following code to it:
+```json
+{
+    "require": {
+        "microsoft/windowsazure": "^0.4"
+    }
+}
+```
+
+* Open a command prompt and execute this in your project root
+
+    php composer install
+
+###Add requirement and install with one command
+Execute the following command in order to get this package:
 
```

Comment:
As I said @yaqiyang, I have installed globally, and I renamed `composer.phar` to just `composer`. It only is one of the proposed practices described at the Composer documentation. Please take into account that it is a PHAR file, so the only difference in my case is that I have it placed (moved) in a path which is part of my environment variables. There is no other process that take place at the tool install.